frappe/desk/page/whats_new_page/whats_new_page.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2022, Frappe Technologies Pvt. Ltd. and Contributors
+# License: MIT. See LICENSE
+
+import frappe
+from frappe.utils.data import get_link_to_form
+import requests
+from frappe import _
+from datetime import datetime
+
+@frappe.whitelist(allow_guest=True)
+def get_whats_new_posts():
+	host = "https://frappe.io"
+	post_list = []
+
+	try:
+		res = requests.get(host + '/api/method/fetch_latest_posts')
```

Comment:
`fetch_latest_posts` is very poor naming. Since this feature is called "What's New", the API method should also reflect that to avoid confusion in the future. Suggestion: `/api/method/fetch-whats-new`